src/main/java/com/example/fizzbuzz/repository/FizzBuzzRepository.java

```diff
@@ -1,7 +1,8 @@
 package com.example.fizzbuzz.repository;
 
 import com.example.fizzbuzz.model.FizzBuzzDB;
-import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.repository.CrudRepository;
 
-public interface FizzBuzzRepository extends JpaRepository<FizzBuzzDB, Long> {
-}
+public interface FizzBuzzRepository extends CrudRepository<FizzBuzzDB, Long> {
```

Comment:
I suggest switching back to JpaRepository
It should provide more methods out of the box

Reply:
Okay, I will do it

Reply:
Done